.idea/encodings.xml

```diff
@@ -12,6 +12,7 @@
     <file url="file://$PROJECT_DIR$/kroxylicious-api/src/main/java" charset="UTF-8" />
```

Comment:
We should be dropping `.idea/encodings.xml` in the same way we are dropping the `iml` files right?